modules/openapi-generator/src/main/resources/kotlin-client/jvm-common/infrastructure/URIAdapter.kt.mustache

```diff
@@ -16,17 +16,17 @@ import com.squareup.moshi.ToJson
 import java.net.URI
 
 {{#moshi}}
-{{#nonPublicApi}}internal {{/nonPublicApi}}class UriAdapter {
+{{#nonPublicApi}}internal {{/nonPublicApi}}class URIAdapter {
```

Comment:
FYI. Android refers to `Uri` instead: https://developer.android.com/reference/kotlin/android/net/Uri

I see that you want to keep it consistent with the file name that I'm totally ok with.